upgrade-steps.md

```diff
@@ -70,4 +70,51 @@ This documents the steps taken to upgrade VPB from ruby 1.93/rails 3.2 to the la
 |app/assets/javascripts/application.js|0%|0% (not needed for rails/ruby upgrade)|
 |app/assets/javascripts/aws-upload.js|0%|0% (not needed for rails/ruby upgrade)|
 
+## Dependency Versions
 
+|gem                      |Environment|Latest |Latest Ruby|Initial 3|Final 3  |Audited/Keep|Audit Notes|
+|-------------------------|-----------|-------|-----------|---------|---------|------------|-----------|
+|capistrano               |all        |3.12.1 |>= 2.0     |2.14.1   |*removed*|---         |Already removed |
+|capistrano-maintenance   |all        |1.2.1  |>= 0       |0.0.2    |*removed*|---         |Already removed |
+|sass-rails               |assets     |6.0.0  |>= 0       |3.2.6    |3.2.6    |Y/N         |No sass files present |
+|coffee-rails             |assets     |5.0.0  |>= 0       |3.2.2    |3.2.2    |Y/N         |No coffee files present |
+|therubyracer             |assets     |0.12.3 |>= 0       |0.12.1   |0.12.1   |Y/N?        |git commmit says its for asset precompile |
+|uglifier                 |assets     |4.2.0  |>= 1.9.3   |1.3.0    |1.3.0    |Y/N?        |not referenced in asset precompile |
+|turbo-sprockets-rails3   |assets     |0.3.14 |>= 0       |0.3.6    |0.3.6    |Y/N?        |deployment will change |
+|jquery-ui-rails          |all        |6.0.1  |>= 0       |4.0.0    |4.0.0    |Y/N?        |Not used in asset pipeline that I can see |
```

Comment:
I suspect that was added in an attempt to bring in jquery-ui without including it in the source. However it looks like it is manually included in the source anyway.